history/peter-toma.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Programmer who developed the oldest commercial machine translation program
+---
+# Peter Toma
+Peter Toma was a computer programmer who designed Systran, the oldest commercial machine translation program. 
+
+He initially developed the program for the United States Air Force. 
+It was intended to translate large-scale scientific documents from Russian into English during the Cold War. 
+
+It was later acquired by the European Commission for English-to-French direction. 
```

Comment:
> The European Commission acquired it for translating from English to French.

Did the acquire the whole thing?  Or did they just become one customer among others?